db/migrate/20250315193051_create_pghero_space_stats.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePgheroSpaceStats < ActiveRecord::Migration[8.0]
+  def change
+    # rubocop:disable Rails/CreateTableWithTimestamps
```

Comment:
There's a `captured_at` timestamp, which I guess is enough. I don't want to depart from PgHero's expected data model.